packages/ngx-flicking/projects/ngx-flicking/src/lib/ngx-flicking.component.ts

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright (c) 2015 NAVER Corp.
+ * egjs projects are licensed under the MIT license
+ */
+import NativeFlicking, { Plugin, FlickingOptions, withFlickingMethods, DEFAULT_OPTIONS, FlickingEvent } from '@egjs/flicking';
+// tslint:disable-next-line: max-line-length
+import { Component, OnInit, Input, AfterViewInit, ElementRef, OnChanges, Output, EventEmitter, OnDestroy, ContentChild, TemplateRef } from '@angular/core';
+import ListDiffer, { DiffResult } from '@egjs/list-differ';
+
+@Component({
+  selector: 'eg-flicking',
```

Comment:
@jongmoon Why don't you use ngx-flickering for the selector?

Reply:
@daybrush I researched angular component and found that **major product** use their namespace as a prefix. 

But I agree with you because product name is `ngx-flicking` and `eg-` would make some confusion.

Reply:
@jongmoon 
I thought the selector was `ngx-flicking` because the class name is `NgxFlicking`. Were the components you were looking for the same class name and selector name?